app/connectors/ChargeHistoryConnector.scala

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import config.FrontendAppConfig
+import models.chargeHistory.{ChargeHistoryResponseModel, ChargesHistoryErrorModel, ChargesHistoryModel}
+import play.api.Logger
+import play.api.http.Status
+import play.api.http.Status.OK
+import uk.gov.hmrc.http.{HeaderCarrier, HttpClient, HttpResponse}
+
+import javax.inject.Inject
+import scala.concurrent.{ExecutionContext, Future}
+
+class ChargeHistoryConnector @Inject()(val http: HttpClient,
```

Comment:
We already have a connector which returns Future[ChargeHistoryResponseModel], see FinancialDetailsConnector.getChargeHistory, so might worth to move method from FinancialDetailsConnector into this new connector for consistency or we can create a tech debt for this ?

Reply:
Hi Pavel, this is that same code, I've just moved it into a new file for clarity